server/src/main/scala/test/controllers/TestController.scala

```diff
@@ -92,6 +94,44 @@ class TestController @Inject()(val config: Configuration) extends InjectedContro
       }
   }
 
+  def addTestArrival(): Action[AnyContent] = Action.async {
```

Comment:
we can probably just replace the old endpoint with this one, seeing as it supports all the existing fields. Do you think we need to keep both?

Reply:
I think , this should be sufficient for both and it would be more about test